bearings:
  f1: [-1.0, 0.0, 0.0]
traj_start: [0.0, 0.0, 0.0]